src/microbe_stage/TweakedProcess.cs

```diff
@@ -50,7 +60,8 @@ public bool Equals(TweakedProcess other)
             return false;
 
         // ReSharper disable once CompareOfFloatsByEqualityOperator
-        return Rate == other.Rate && ReferenceEquals(Process, other.Process);
+        return Rate == other.Rate && SpeedMultiplier == other.SpeedMultiplier
+            && ReferenceEquals(Process, other.Process);
```

Comment:
I think I was able to confirm that this change breaks the ProcessStatistics.GetAndMarkUsed so I will revert this change. (this now causes process panel process order to change when turning processes off and on)